src/tools/pmempool/common.c

```diff
@@ -1450,8 +1450,9 @@ pool_set_file_open(const char *fname,
  * pool_set_file_close -- closes pool set file or regular file
  */
 void
-pool_set_file_close(struct pool_set_file *file)
+pool_set_file_close(struct pool_set_file **file_)
```

Comment:
Is it really necessary ? The free(file) was missing obviously but why you would like to set the pointer to NULL ?